### ISSUE TITLE
Refactor python function to create commandline arguments

### ISSUE THREAD

**Issue by larsbergstrom** [servo/saltfs#338]
In https://github.com/servo/saltfs/pull/331, we added a function to create the path string, `make_cd_string`:
https://github.com/servo/saltfs/blob/master/buildbot/master/files/config/factories.py#L118-L119

However, this could be cleaner! As seen in https://github.com/servo/saltfs/blob/master/buildbot/master/files/config/factories.py#L126-L127, there are more arguments in the array that are static.

In this fix, please rename the function and make it return an array with *all* of the arguments required, while still taking the same single argument (the command to execute).

**Comment by ashrko619** [comment]
Are you suggesting something like this ?
```python
def make_cd_command(command):
    return ["bash", "-l", "-c", "cd /c/buildbot/slave/windows/build ; " + command ]
	
windows = ServoFactory([
    # TODO: convert this to use DynamicServoFactory
    # We need to run each command in a bash login shell, which breaks the
    # heuristics used by DynamicServoFactory.make_step
    steps.Compile(command= make_cd_command("./mach build -d -v"),
                  env=envs.build_windows),
    steps.Compile(command= make_cd_command("./mach test-unit"),
                  env=envs.build_windows),

    # TODO: run lockfile_changed.sh and manifest_changed.sh scripts
])
```

**Comment by aneeshusa** [comment]
@ashrko619 Yes, that's the right direction. If you open a PR I have a few style nits, but it looks good otherwise.

**Comment by ashrko619** [comment]
@aneeshusa Thanks, I will open a PR, btw what are the style nits here ?

**Comment by aneeshusa** [comment]
Here, we need to rename the `make_cd_command` function to reflect its updated scope, and there are some extra spaces.

Generally, I prefer to do reviewing on PRs because inline comments are clearer, so go ahead and open a PR and I'll make any future comments there.

**Comment by aneeshusa** [comment]
Fixed by #342.
